lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,325 @@
+---
+name: Financial Transaction
+field_name_base_url: http://hl7.org/fhir/r4/Basic
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType":"Basic"
+    }
+  note: resourceType must be Basic
+  url: http://hl7.org/fhir/r4/json.html#resources
+
+- name: identifier
+  required: 'Yes'
+  cardinality: 0..*
+  type: Identifier
+  description: Identifier assigned to the resource for business purposes.
+  action: create
+  example: |
+    {
+      "identifier": [
+        {
+            "value": "33358000",
+            "use": "usual"
+        }
+      ]
+    }
+  note: At least one external transaction identifier must be provided.
+  url: http://hl7.org/fhir/basic-definitions.html#Basic.identifier
+
+- name: code
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: Identifies the 'type' of resource - equivalent to the resource name for other resources.
+  action: create
+  example: |
+    {
+      "code": "fintrans"
```

Comment:
This needs to be updated

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/a1a39411fbb35b2ccf79e521ffae5e6ae92d62b9#diff-26870e1b94fc9955e130f3f940dbe2e7R44

Reply:
The structure needs to be a CodeableConcept

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/e78e19c1c6ca51190af53b4e0d196efefff0439b#diff-26870e1b94fc9955e130f3f940dbe2e7R43